### PR TITLE
Multiple photos & iPhone X support

### PR DESCRIPTION
**Pull request by davea**
Further bits broken out of the Ground Control branch. Mainly multiple photos and iPhone X support.

### CODE REVIEW

File: www/css/fms.css [hunk 1]
```diff
@@ -161,37 +161,36 @@
     }
 
 /* ios 7 tweaks to deal with different status bar behaviour */
-    .ios7 .ui-mobile-viewport {
```
[the remark this answers is not in the document]
Comment (davea) [reply www/css/fms.css]: Yep, that's right. There's no need for any iOS-specific behaviour in `map-OpenLayers.js` any more.

**Comment by davea** [comment]
Thanks for the review, and for spotting that the offline stuff was entirely broken! I think it's all ship-shape and ready for another look now.

File: www/js/files.js [hunk 2]
```diff
@@ -57,6 +57,20 @@
 
             },
 
+            deleteURIs: function(uris) {
+                console.log("deleteURIs", uris);
+                var deferred = null;
```
Comment (dracos) [inline www/js/files.js]: If you put `var deferred = Promise.resolve();` here (think that's right?), then you wouldn't need the `if` inside the loop.

File: www/js/views/offline.js [hunk 3]
```diff
@@ -39,11 +40,21 @@
 
             afterDisplay: function() {
                 $('body')[0].scrollTop = 0;
-                $('div[data-role="content"]').show();
+
+                // The height of the photos container needs to be adjusted
+                // depending on the number of photos - if the maximum number if
```
Comment (dracos) [inline www/js/views/offline.js]: Typo in comment.

File: www/js/views/offline.js [hunk 4]
```diff
@@ -73,51 +93,72 @@
...
-
-                    $('#photo-next-btn .ui-btn-text').text(FMS.strings.next);
-                    $('#display_photo').show();
-                    $('#add_photo').hide();
+                    var files = that.model.get('files');
+                    files.push(file.toURL());
+                    that.model.set('files', files);
+                    that.updateCurrentReport();
+
+                    // window.setTimeout(function() {
```
Comment (dracos) [inline www/js/views/offline.js]: I see the timeout removed from photo.js, not needed here either?

**Pull request merged**